### PR TITLE
Add pagination to series secondary tags

### PR DESCRIPTION
**Pull request by davidthomas22**
### Context

> Does this issue have a Trello card?

https://trello.com/c/w9ODRIv3

### Intent

> What changes are introduced by this PR that correspond to the above card?

When showing the full results of a secondary tag or series, paginate them to allow more than 40 results to be shown (the current limit)

> Would this PR benefit from screenshots?

### Considerations

> Is there any additional information that would help when reviewing this PR?

> Are there any steps required when merging/deploying this PR?

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/routes/__tests__/tags.spec.js [hunk 1]
```diff
@@ -144,4 +133,101 @@ describe('GET /tags', () => {
...
+      describe('tags page JSON endpoint', () => {
+        it('correctly returns the JSON response', () => {
+          const cmsService = {
+            getPage: jest.fn().mockReturnValue(data),
+          };
+          const router = createTagRouter({ cmsService });
+          const app = setupBasicApp();
+
+          app.use(sessionMiddleware);
+          app.use('/tags', router);
```
Comment (jonhindle) [inline server/routes/__tests__/tags.spec.js:199]: this is duplication(212). Probably best in a beforeEach.

File: server/routes/__tests__/tags.spec.js [hunk 2]
```diff
@@ -144,4 +133,101 @@ describe('GET /tags', () => {
       });
     });
   });
+
+  describe('/:id.json', () => {
+    describe('on error', () => {
+      beforeEach(() => {
+        jest.clearAllMocks();
+      });
```
Comment (jonhindle) [inline server/routes/__tests__/tags.spec.js:141]: This is a catchall and probably could be on the parent _describe_ so it applies to all tests.

File: server/routes/tags.js [hunk 3]
```diff
@@ -3,6 +3,21 @@ const express = require('express');
 const createTagRouter = ({ cmsService }) => {
   const router = express.Router();
 
+  router.get('/:id.json', async (req, res, next) => {
+    try {
+      const { id } = req.params;
+      const { page } = req.query;
```
Comment (jonhindle) [inline server/routes/tags.js:9]: Personal choice - could single line it 😬 
```
const { params: { id }, query: { page }, session: { establishmentName } } = req;
```

**Review by jonhindle — APPROVED** [review]
Minor comments. Nothing show stopping though.

**Review by andrewrlee — APPROVED** [review]
LGTM - going to address @jonhindle comments in next PR

**Pull request merged**